examples/registration.rb

```diff
@@ -0,0 +1,73 @@
+# This program provides a simple example illustrating how to programmatically
+# register a client with InnoVault and e3db. In some situations, it's preferable
+# to register a client from the server or system that will be using its
+# credentials (to ensure that all data is truly encrypted from end-to-end
+# with no possibilities of a credential leak). For more detailed information,
```

Comment:
> In some situations, it's preferable to register a client...

This section makes me think that credential leaks just happen in other flows. I'd reword or omit.